test/snippet/range/views/minimiser.cpp

```diff
@@ -18,4 +19,12 @@ int main()
 
     // kmer_hash with gaps, hashes: [2,7,8,14,1], minimiser: [2,1]
     seqan3::debug_stream << (text | seqan3::views::kmer_hash(0b101_shape) | seqan3::views::minimiser(4)) << '\n';
+
+    // Minimiser view with two ranges
+    auto hashes2 = text | seqan3::views::complement | std::views::reverse
+                        | seqan3::views::kmer_hash(seqan3::ungapped{3}) | std::views::reverse;
```

Comment:
You reverse twice;

so you hash the reverse complement, but because now you need to compare the begin of the forward hashes with the end of the reverse complement hashes.
And that's why you reverse again to have the correct order, right?

This needs to be well explained.

Reply:
I think the design, as it is now, is not really intuitive. If you need to document in which order you have to provide the input, this does not seem to be straight-forward and user-friendly.

I added this PR to the strategy meeting, because I think that we could have a much cleaner design here.

Reply:
Well, it is complicated for the specific use case but if I understand the minimiser view correctly it is not aware of the what to calculate the minimum on. So it could be that I want to compute the minimisers of two unrelated sequences. This case just demonstrates how to do it for the reverse complement of the same sequence. I am not sure, but @smehringer mentioned a dedicated view adaptor that does that internally? 
I would adapt the snippet like so:

```cpp
auto hashes2 = text | reverse_complement // get the reverse complement.
                    | seqan3::views::kmer_hash(seqan3::ungapped{3}) // transform into hash sequence.
                    | std::views::reverse; // reverse to preserve order of the hashes with the forward sequence.
```